{"level":"error","message":"Uncaught Exception thrown: Oops!Oops!","stack":"Error: Oops!\n    at Object.<anonymous> (/Users/oc42ov/projects/nodejs-course/src/server.ts:17:7)\n    at Module._compile (internal/modules/cjs/loader.js:1158:30)\n    at Module.m._compile (/Users/oc42ov/projects/nodejs-course/node_modules/ts-node/src/index.ts:836:23)\n    at Module._extensions..js (internal/modules/cjs/loader.js:1178:10)\n    at Object.require.extensions.<computed> [as .ts] (/Users/oc42ov/projects/nodejs-course/node_modules/ts-node/src/index.ts:839:12)\n    at Module.load (internal/modules/cjs/loader.js:1002:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:901:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)\n    at main (/Users/oc42ov/projects/nodejs-course/node_modules/ts-node/src/bin.ts:226:14)\n    at Object.<anonymous> (/Users/oc42ov/projects/nodejs-course/node_modules/ts-node/src/bin.ts:485:3)","label":"server.ts","timestamp":"2020-04-12 22:52:18"}
{"level":"error","message":"Unhandled Rejection at PromiseOops!","stack":"Error: Oops!\n    at Object.<anonymous> (/Users/oc42ov/projects/nodejs-course/src/server.ts:18:16)\n    at Module._compile (internal/modules/cjs/loader.js:1158:30)\n    at Module.m._compile (/Users/oc42ov/projects/nodejs-course/node_modules/ts-node/src/index.ts:836:23)\n    at Module._extensions..js (internal/modules/cjs/loader.js:1178:10)\n    at Object.require.extensions.<computed> [as .ts] (/Users/oc42ov/projects/nodejs-course/node_modules/ts-node/src/index.ts:839:12)\n    at Module.load (internal/modules/cjs/loader.js:1002:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:901:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)\n    at main (/Users/oc42ov/projects/nodejs-course/node_modules/ts-node/src/bin.ts:226:14)\n    at Object.<anonymous> (/Users/oc42ov/projects/nodejs-course/node_modules/ts-node/src/bin.ts:485:3)","label":"server.ts","timestamp":"2020-04-12 22:55:31"}
